content/pt/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md

```diff
@@ -0,0 +1,129 @@
+---
+reviewers:
+- femrtnz
+- jcjesus
```

Comment:
I Believe we should keep the original reviews here, as this is related to the content, not the translation.

cc @remyleone